runtimes/eden/Cargo.toml

```diff
@@ -50,7 +50,7 @@ std = [
   "cumulus-pallet-dmp-queue/std",
   "cumulus-pallet-parachain-system/std",
   "cumulus-pallet-xcm/std",
-  #"cumulus-pallet-xcmp-queue/std",
+  "cumulus-pallet-xcmp-queue/std",
```

Comment:
This pallet should be listed under "try-runtime" with `cumulus-pallet-xcmp-queue/try-runtime` as well.

Reply:
Also it should be listed under "runtime-benchmarks" with `cumulus-pallet-xcmp-queue/runtime-benchmarks`.

Reply:
Done